docs/release-notes/remove-hp-importance.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Removed Features**
+
+-  "Hyperparameter importance" feature and associated API endpoints have been removed.
```

Comment:
FWIW I've definitely seen one user who was using this API directly for something. I'll see if I can dig up the Slack context as a final check that we shouldn't keep this for some reason other than the drop-down sorting.

Reply:
This might have been it: https://determined-community.slack.com/archives/CV37XVAGL/p1632325977028600. I seem to recall a longer thread too.

Reply:
this user is no longer our user, unfortunately, so don't have to notify them.